services/FeefoReviewService.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Craft;
+
+class FeefoReviewService extends BaseApplicationComponent
+{
+  public $plugin = null;
+  public $settings = array();
+
+  public function __construct()
+  {
+    $this->plugin   = craft()->plugins->getPlugin('feeforeview');
+    $this->settings = $this->plugin->getSettings();
+  }
+
+  public function getReviewSummary($id)
```

Comment:
Not neccessary for at this time but we could pass in variables to this from the views for some of the hardcoded parameters that are sent with the product reviews request

Reply:
For now, it should be okay as we have another global setting page for the plugin.